sql/core/src/test/resources/sql-tests/results/postgreSQL/numeric.sql.out

```diff
@@ -4654,111 +4654,111 @@ struct<id:bigint>
 -- !query
 select ln(1.2345678e-28)
 -- !query schema
-struct<LOG(1.2345678E-28):double>
+struct<log(1.2345678E-28):double>
```

Comment:
Can we have `ln` instead of `log`?

Reply:
I actually realised that there are some more instances such as `ToDegrees`, `ToRadians`, `UnaryMinus` and `UnaryPositive`.. let me exclude this these to make the scope my PR address smaller